### PR TITLE
client/http: add func for jenkins test

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Ref #7300

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: client/http/api.go [hunk 1]
```diff
@@ -38,6 +38,9 @@ const (
 	store                     = "/pd/api/v1/store"
 	Stores                    = "/pd/api/v1/stores"
 	StatsRegion               = "/pd/api/v1/stats/region"
+	MembersPrefix             = "/pd/api/v1/members"
+	LeaderPrefix              = "/pd/api/v1/leader"
+	TransferLeader            = "/pd/api/v1/leader/transfer"
```
Comment (JmPotato) [inline client/http/api.go:43]: If these two consts will always be used with the exported functions below, it's better to keep them unexported.

```suggestion
	leaderPrefix              = "/pd/api/v1/leader"
	transferLeader          = "/pd/api/v1/leader/transfer"
```

File: client/http/client.go [hunk 2]
```diff
@@ -458,6 +466,44 @@ func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRan
 	return &regionStats, nil
 }
 
+// SetStoreLabels sets the labels of a store.
+func (c *client) SetStoreLabels(ctx context.Context, storeID int64, storeLabels map[string]string) error {
+	jsonBody, err := json.Marshal(storeLabels)
```
Comment (JmPotato) [inline client/http/client.go:471]: Better align with the naming style of other functions.

```suggestion
	jsonInput, err := json.Marshal(storeLabels)
```

File: client/http/client.go [hunk 3]
```diff
@@ -458,6 +466,44 @@ func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRan
...
+	if err != nil {
+		return errors.Trace(err)
+	}
+	return c.requestWithRetry(ctx, "SetStoreLabel", LabelByStoreID(storeID),
+		http.MethodPost, bytes.NewBuffer(jsonBody), nil)
+}
+
+func (c *client) GetMembers(ctx context.Context) (*MembersInfo, error) {
+	members := MembersInfo{}
```
Comment (JmPotato) [inline client/http/client.go:480]: Ditto.

```suggestion
	var members MembersInfo
```

File: tests/integrations/client/http_client_test.go [hunk 4]
```diff
@@ -384,3 +384,63 @@ func (suite *httpClientTestSuite) TestScheduleConfig() {
 	re.Equal(float64(8), config["leader-schedule-limit"])
 	re.Equal(float64(2048), config["region-schedule-limit"])
 }
+
+func (suite *httpClientTestSuite) TestSchedulers() {
+	re := suite.Require()
+	schedulers, err := suite.client.GetSchedulers(suite.ctx)
+	re.NoError(err)
+	re.Len(schedulers, 0)
```
Comment (JmPotato) [inline tests/integrations/client/http_client_test.go:392]: Why is it zero? Shouldn't it be equal to the default number of schedulers?

Comment (HuSharp) [reply tests/integrations/client/http_client_test.go]: Because the test doesn't wait for the coordinator to start

File: client/http/client.go [hunk 5]
```diff
@@ -458,6 +466,44 @@ func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRan
...
+	if err != nil {
+		return nil, err
+	}
+	return &members, nil
+}
+
+// GetLeader gets the leader of PD cluster.
+func (c *client) GetLeader(context.Context) (*pdpb.Member, error) {
+	var leader pdpb.Member
+	err := c.requestWithRetry(context.Background(), "GetLeader", leaderPrefix,
```
Comment (JmPotato) [inline client/http/client.go:493]:
```suggestion
func (c *client) GetLeader(ctx context.Context) (*pdpb.Member, error) {
	var leader pdpb.Member
	err := c.requestWithRetry(ctx, "GetLeader", leaderPrefix,
```

File: tests/integrations/client/http_client_test.go (hunk 4 above)
Comment (HuSharp) [reply tests/integrations/client/http_client_test.go:392]: Maybe we don't need to wait for the coordinator to start working, just focus on the functionality of HTTP client

**Review by JmPotato — APPROVED** [review]

File: pkg/schedule/handler/handler.go [hunk 6]
```diff
@@ -808,7 +808,7 @@ func (h *Handler) GetSchedulerByStatus(status string, needTS bool) (interface{},
 	switch status {
 	case "paused":
 		pausedSchedulers := make([]string, 0, len(schedulers))
-		pausedPeriods := []schedulerPausedPeriod{}
```
Comment (lhy1024) [inline pkg/schedule/handler/handler.go:811]: it will return nil rather than `[]` when it is empty, which will make some qa tests failed.

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

File: pkg/schedule/handler/handler.go (hunk 6 above)
Comment (HuSharp) [reply pkg/schedule/handler/handler.go]: get it. fixed.

**Comment by HuSharp** [comment]
@JmPotato @lhy1024 can you leave a merge label? thx! :)

**Pull request merged**